app/src/main/java/com/readtracker/android/activities/ImportActivity.java

```diff
@@ -30,21 +35,23 @@
   private static final String TAG = ImportActivity.class.getSimpleName();
```

Comment:
Small little note: in some classes you use ```class.getSimpleName()``` - like here - and in other you use ```class.getName()``` - like in the SettingsActivity f.x.